include/revng/Support/DecompilationHelpers.h

```diff
@@ -117,3 +117,41 @@ inline const llvm::CallInst *isCallToNonIsolated(const llvm::Value *I) {
 inline bool isHelperAggregateLocalVarDecl(const llvm::Value *I) {
   return isCallToNonIsolated(I) and I->getType()->isAggregateType();
 }
+
+inline bool isStatement(const llvm::Instruction &I) {
```

Comment:
@pfez can you confirm that this is the One True Way™ to say if an `Instruction` is a statement or not, in the context of the old backend after `switch-to-statements`?